[
    {
        "id": "app.error.reportErrorTitle",
        "description": "title for report error modal",
        "defaultMessage": "Report error"
    },
    {
        "id": "app.error.thankYouForYourFeedback",
        "description": "thank you for feedback",
        "defaultMessage": "شكرا على ملاحظاتك!"
    },
    {
        "id": "app.error.copyTheFollowingLines",
        "description": "text explaining what to do to report an issue on GH",
        "defaultMessage": "Please copy the following lines and create a new issue on {githubLink}."
    },
    {
        "id": "app.error.alsoCheckForOpenedIssues",
        "description": "text suggesting to check for existing issues",
        "defaultMessage": "Also check if there are no already opened issues regarding this error!"
    },
    {
        "id": "app.error.backupKeys",
        "description": "error message for backup keys action",
        "defaultMessage": "لقد فشل توليد النسخ الاحتياطي. حاول مرة أخرى"
    },
    {
        "id": "app.error.commentsDownvote",
        "description": "error message for downvoting a comment",
        "defaultMessage": "حدث خطأ أثناء التصويت السلبي للتعليق"
    },
    {
        "id": "app.error.commentsPublish",
        "description": "error message for publishing a comment",
        "defaultMessage": "حدث خطأ أثناء نشر التعليق"
    },
    {
        "id": "app.error.commentsUpvote",
        "description": "error message for upvoting a comment",
        "defaultMessage": "حدث خطأ أثناء التصويت الإجابي للتعليق"
    },
    {
        "id": "app.error.dashboardAdd",
        "description": "error message for creating a new board",
        "defaultMessage": "حدث خطأ أثناء حفظ اللوحة الجديدة"
    },
    {
        "id": "app.error.dashboardAddColumn",
        "description": "error message for adding a column to a board",
        "defaultMessage": "حدث خطأ أثناء حفظ العمود الجديد"
    },
    {
        "id": "app.error.dashboardDelete",
        "description": "error message for deleting a board",
        "defaultMessage": "حدث خطأ أثناء حذف لوحتك"
    },
    {
        "id": "app.error.dashboardDeleteColumn",
        "description": "error message for removing a column from a board",
        "defaultMessage": "حدث خطأ أثناء حذف عمودك"
    },
    {
        "id": "app.error.dashboardGetActive",
        "description": "error message for fetching the active board",
        "defaultMessage": "حدث خطأ أثناء الحصول على اللوحة النشطة"
    },
    {
        "id": "app.error.dashboardGetAll",
        "description": "error message for fetching all boards",
        "defaultMessage": "حدث خطأ أثناء الحصول على لوحاتك"
    },
    {
        "id": "app.error.dashboardSetActive",
        "description": "error message for setting the active board",
        "defaultMessage": "حدث خطأ أثناء حفظ لوحتك النشطة"
    },
    {
        "id": "app.error.dashboardUpdateColumn",
        "description": "error message for updating a column",
        "defaultMessage": "حدث خطأ أثناء تحديث عمودك"
    },
    {
        "id": "app.error.draftAutosave",
        "description": "error message for autosaving a draft",
        "defaultMessage": "لا يمكن تحفظ التغييرات الأخيرة على هذه المسودة"
    },
    {
        "id": "app.error.draftPublish",
        "description": "error message for publishing a draft",
        "defaultMessage": "حدث خطأ أثناء نشر مسودتك"
    },
    {
        "id": "app.error.entriesGetAsDrafts",
        "description": "error message for fetching your own entries",
        "defaultMessage": "حدث خطأ أثناء محاولة جلب مداخلاتك. عرض المسودات المحلية فقط"
    },
    {
        "id": "app.error.entryClaim",
        "description": "error message for collecting Essence for an entry",
        "defaultMessage": "حدث خطأ أثناء جميع Essence ل {entryTitle}"
    },
    {
        "id": "app.error.entryClaimVote",
        "description": "error message for collecting Essence for a vote",
        "defaultMessage": "حدث خطأ أثناء جميع Essence ل {entryTitle}"
    },
    {
        "id": "app.error.entryDownvote",
        "description": "error message for downvoting an entry",
        "defaultMessage": "حدث خطأ أثناء التصويت سلبيا ل {entryTitle}"
    },
    {
        "id": "app.error.entryGetFull",
        "description": "error message for fetching a full entry",
        "defaultMessage": "حدث خطأ أثناء الحصول على بيانات المداخلة"
    },
    {
        "id": "app.error.entryListIterator",
        "description": "error message for fetching entries in a list",
        "defaultMessage": "حدث خطأ أثناء الحصول على المداخلات في القائمة"
    },
    {
        "id": "app.error.entryUpvote",
        "description": "error message for upvoting an entry",
        "defaultMessage": "حدث خطأ أثناء التصويت إجابيا ل {entryTitle}"
    },
    {
        "id": "app.error.errorNotificationTitle",
        "description": "title for error notifcations",
        "defaultMessage": "رمز الخطأ {errorCode}"
    },
    {
        "id": "app.error.fatalError",
        "description": "title for fatal error modal",
        "defaultMessage": "خطأ فادح"
    },
    {
        "id": "app.error.generalSettings",
        "description": "error message for fetching general settings",
        "defaultMessage": "حدث خطأ أثناء الحصول على إعداداتك العامة"
    },
    {
        "id": "app.error.gethGetOptions",
        "description": "error message for fetching geth options",
        "defaultMessage": "حدث خطأ أثناء الحصول على خيارات خدمة geth"
    },
    {
        "id": "app.error.gethGetStatus",
        "description": "error message for fetching geth status",
        "defaultMessage": "حدث خطأ أثناء الحصول على حالة و وضع خدمة geth"
    },
    {
        "id": "app.error.gethGetSyncStatus",
        "description": "error message for fetching geth synchronization status",
        "defaultMessage": "حدث خطأ أثناء الحصول على حالة تزامن"
    },
    {
        "id": "app.error.gethSaveSettings",
        "description": "error message for saving geth settings",
        "defaultMessage": "حدث خطأ أثناء حفظ إعدادات geth"
    },
    {
        "id": "app.error.gethSettings",
        "description": "error message for fetching geth settings",
        "defaultMessage": "حدث خطأ أثناء الحصول على إعدادات geth الخاصة بك"
    },
    {
        "id": "app.error.gethStart",
        "description": "error message for starting geth service",
        "defaultMessage": "لا يمكن بدء خدمة Geth"
    },
    {
        "id": "app.error.gethStop",
        "description": "error message for stopping geth service",
        "defaultMessage": "لا يمكن إيقاف خدمة Geth"
    },
    {
        "id": "app.error.ipfsGetConfig",
        "description": "error message for fetching IPFS service configuration",
        "defaultMessage": "حدث خطأ أثناء الحصول على تهيئة خدمة IPFS"
    },
    {
        "id": "app.error.ipfsGetPorts",
        "description": "error message for fetching IPFS ports",
        "defaultMessage": "حدث خطأ أثناء الحصول على منافذ خدمة IPFS"
    },
    {
        "id": "app.error.ipfsSaveSettings",
        "description": "error message for saving IPFS settings",
        "defaultMessage": "حدث خطأ أثناء حفظ إعدادات IPFS الخاصة بك"
    },
    {
        "id": "app.error.ipfsSetPorts",
        "description": "ipfs set ports error message",
        "defaultMessage": "حدث خطأ أثناء ضبط منافذ خدمة ipfs"
    },
    {
        "id": "app.error.ipfsSettings",
        "description": "error message for fetching IPFS settings",
        "defaultMessage": "حدث خطأ أثناء الحصول على إعدادات IPFS الخاصة بك"
    },
    {
        "id": "app.error.ipfsStart",
        "description": "error message for starting IPFS service",
        "defaultMessage": "تعذر بدء خدمة IPFS"
    },
    {
        "id": "app.error.ipfsGetStatus",
        "description": "error message for fetching IPFS service status",
        "defaultMessage": "حدث خطأ أثناء الحصول على حالة و وضع خدمة IPFS"
    },
    {
        "id": "app.error.ipfsStop",
        "description": "error message for stopping IPFS service",
        "defaultMessage": "تعذر إيقاف خدمة IPFS"
    },
    {
        "id": "app.error.unexpectedError",
        "description": "a general error (with no error code)",
        "defaultMessage": "خطأ غير متوقع"
    },
    {
        "id": "app.error.profileBondAeth",
        "description": "error message for bonding (manafying) AETH",
        "defaultMessage": "حدث خطأ أثناء تحويل {amount} (أكاشاإثير) AETH إلى مانا"
    },
    {
        "id": "app.error.profileCycleAeth",
        "description": "error message for cycling AETH",
        "defaultMessage": "حدث خطأ أثناء تدوير {amount} (أكاشاإثير) AETH"
    },
    {
        "id": "app.error.profileFreeAeth",
        "description": "error message for freeing AETH that was cycled",
        "defaultMessage": "حدث خطأ أثناء جمع (أكاشاإثير) AETH المدورة و الخاصة بك"
    },
    {
        "id": "app.error.profileGetLocal",
        "description": "error message for fetching local profiles",
        "defaultMessage": "حدث خطأ أثناء جلب ملفات التشكيلات المحلية"
    },
    {
        "id": "app.error.profileGetList",
        "description": "error message for fetching a list of profiles",
        "defaultMessage": "حدث خطأ أثناء الحصول على قائمة ملفات التعريف"
    },
    {
        "id": "app.error.profileToggleDonations",
        "description": "error message for changing the settings for accepting tips",
        "defaultMessage": "حدث خطأ أثناء تغيير إعدادات tips الخاصة بك"
    },
    {
        "id": "app.error.profileTransferAeth",
        "description": "error message for transferring AETH",
        "defaultMessage": "حدث خطأ أثناء نقل {tokenAmount} (أكاشاإثير) AETH"
    },
    {
        "id": "app.error.profileTransferEth",
        "description": "error message for transferring ETH",
        "defaultMessage": "حدث خطأ أثناء نقل {value} (إثير) ETH"
    },
    {
        "id": "app.error.profileTransformEssence",
        "description": "error message for transforming Essence in AETH",
        "defaultMessage": "حدث خطأ أثناء تحويل {amount} من Essence"
    },
    {
        "id": "app.error.profileUpdate",
        "description": "error message for updating your profile",
        "defaultMessage": "حدث خطأ أثناء تحديث ملف التعريف الخاص بك"
    },
    {
        "id": "app.error.saveGeneralSettings",
        "description": "error message for saving general settings",
        "defaultMessage": "حدث خطأ أثناء حفظ الإعدادات الخاصة بك"
    },
    {
        "id": "app.error.appErrorTitle",
        "description": "Title message to display when the app crashes because of an unexpected error",
        "defaultMessage": "آه مفاجئة!"
    },
    {
        "id": "app.error.appErrorSubtitle",
        "description": "Subtitle message to display when the app crashes because of an unexpected error",
        "defaultMessage": "حدث خطأ فظيع."
    },
    {
        "id": "app.error.copyError",
        "description": "label for copy to clipboard button",
        "defaultMessage": "نسخ الخطأ في الحافظة"
    },
    {
        "id": "app.error.profileFaucet",
        "description": "error message for faucet request",
        "defaultMessage": "حدث خطأ أثناء طلب أموال من الحنفية"
    },
    {
        "id": "app.error.profileRegister",
        "description": "error message for registering your profile",
        "defaultMessage": "حدث خطأ أثناء تسجيل ملف التعريف الخاص بك"
    },
    {
        "id": "app.error.profileFollow",
        "description": "error message for profile follow",
        "defaultMessage": "حدث خطأ أثناء متابعة {ethAddress}"
    },
    {
        "id": "app.error.profileUnfollow",
        "description": "error message for profile unfollow",
        "defaultMessage": "حدث خطأ أثناء إلغاء متابعة {ethAddress}"
    }
]